cql3/values.hh

```diff
@@ -265,6 +268,9 @@ public:
         }, std::move(_data));
     }
     raw_value_view to_view() const;
+    raw_value_view view() const {
```

Comment:
To avoid future confusion, consider doing it the other way round, and renaming `to_view()` to `view()`. It looks like there are 8 trivial callers of this function right now, so it looks manageable.

Reply:
I can follow up with elimination of to_view() (or of view(), it doesn't really matter).

Reply:
Alright, I'll proceed with queuing this one